src/schema/submission_centric_schema.yaml

```diff
@@ -1085,6 +1108,7 @@ slots:
     inlined: true
```

Comment:
data use permission is not defined. I think we lost it during splitting, because I cannot find it in none of our schemes.